.github/workflows/unit-tests.yml

```diff
@@ -8,7 +8,13 @@ on:
   pull_request:
     paths:
       - '!docs/**'
```

Comment:
I guess we should include .github too?

Reply:
@Itxaka had concerns, so I removed it.

Reply:
but if we don't include it among the `paths`, any modification to it won't trigger a workflow build, no?